code/front-end/Satellite GUI.md

```diff
@@ -9,27 +9,24 @@ There are two ways to start local development environment depending on what you
 ### Prerequisites
 
 - Clone the [storj repo](https://github.com/storj/storj)
-- Make sure you have both [node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/get-npm) installed
+- Make sure you have [Go](https://go.dev/doc/install), [node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/get-npm) installed
```

Comment:
Let's mention what Go and NodeJS version to install referencing to have a look at the Makefile of the project.

See https://review.dev.storj.io/c/storj/storj/+/6660 which adds a variable for the NodeJS version.

Reply:
Thanks! Done

Reply:
I don't know if it's a good idea to reference the Makefile lines because if we make some changes on it and those lines move, we likely forget to update this document.

I think it's better to mention to have a look at the Makefile in the first lines of it and link to the makefile.

Reply:
done! Thanks